core/src/test/resources/test_assets.yaml

```diff
@@ -12,6 +12,9 @@ assets:
       enabled: true
       min_amount: 1
       max_amount: 10000
+      types:
```

Comment:
It seems a lot of efforts to maintain both `json` and `yaml`. 
💡 I think we can have the `yaml` for asset configuration testing and `json` just to test the format.